core/src/main/java/org/fao/geonet/kernel/SchemaManager.java

```diff
@@ -1742,8 +1757,7 @@ private boolean isMatchingAttributeInMetadata(Attribute needle, Element haystack
      * @param haystack the XML metadata record we are searching
      */
     private boolean isMatchingNamespaceInMetadata(Namespace needle, Element haystack) {
-        if (Log.isDebugEnabled(Geonet.SCHEMA_MANAGER))
-            Log.debug(Geonet.SCHEMA_MANAGER, "Matching " + needle.toString());
+        LOGGER.debug(Geonet.SCHEMA_MANAGER_MARKER, "Matching {}", needle);
```

Comment:
Is not required to call `needle.toString()`?

Reply:
No it is not, the default is to call toString()